hugo/content/faqs/_index.md

```diff
@@ -2,87 +2,88 @@
 title: Frequently Asked Questions
 ---
 
-[faq]
-title = "FAQs"
+### FAQs
 
-## About the Genome Portal
+#### About the Genome Portal
 
-[[faq.section]]
-question = "What is the Swedish Reference Genome Portal?"
-answer = "TBD"
+{{< faq_block title="How does the Swedish Reference Genome Portal differ from other genome portal initiatives? (e.g., Ensembl, UCSC)" >}}
+The Swedish Reference Genome Portal (SRGP) does not replace existing genome portal initiatives, as it has specific aims and scope:
 
-[[faq.section]]
-question = "How does the Genome Portal differ from existing genome browsers such as Ensembl, USCS, JGI, IGV, among others?"
-answer = "TBD"
+- The SRGP targets the Swedish research community and is restricted to non-human eukaryotic species. Therefore, the species displayed here are typically limited to those submitted by researchers affiliated with Swedish institutions. In contrast, global genome portals such as Ensembl and UCSC have much broader taxonomic scope, aiming to include species from around the world, including humans and bacteria.
```

Comment:
Should we state somewhere that the Genome Portal is not a repository? I think that is something we have often talked about with researchers.

I also suggest to change the word `bacteria` to `prokaryotes` to include that we do not accept archaea.

```suggestion
The SRGP targets the Swedish research community and is restricted to non-human eukaryotic species. Therefore, the species displayed here are typically limited to those submitted by researchers affiliated with Swedish institutions. In contrast, global genome portals such as Ensembl and UCSC have much broader taxonomic scope, aiming to include species from around the world, including humans and prokaryotes.

Reply:
Thank you @brinkdp, commit suggestion done.